conans/client/command.py

```diff
@@ -65,6 +65,14 @@ def __init__(self, conan_api, client_cache, user_io, outputer):
         self._user_io = user_io
         self._outputer = outputer
 
+    def clean(self, *args):
```

Comment:
Do both things (clear the cache and the test_package) without a parameter feels too broad.
Maybe a `conan clean .` or `conan clean path` could clean the local one and `conan clean` clear the local cache.